src/strategies/planet-finance-v2/index.ts

```diff
@@ -0,0 +1,208 @@
+
+const aquaInfinityAbi = [
+  'function getUserGtokenBal(address) view returns (uint256)'
+]
+
+const aquaLendingAbi = [
+  'function getAccountSnapshot(address) view returns (uint256,uint256,uint256,uint256)'
+];
+
+const planetFinanceFarmContractAddress =
+  '0x0ac58Fd25f334975b1B61732CF79564b6200A933';
+
+const gammaFarmAddress = '0xB87F7016585510505478D1d160BDf76c1f41b53d';
+
+const aquaAddress = '0x72B7D61E8fC8cF971960DD9cfA59B8C829D91991';
+
+const aquaBnbLpTokenAddress = '0x03028D2F8B275695A1c6AFB69A4765e3666e36d9';
+
+const aquaGammaLpTokenAddress = '0xcCaF3fcE9f2D7A7031e049EcC65c0C0Cc331EE0D';
+
+const aquaLendingAddress = '0xb7eD4A5AF620B52022fb26035C565277035d4FD7';
+
+const aquaInfinityAddress = '0x6E7a174836b2Df12599ecB2Dc64C1F9e1576aC45';
+
+const increase_in_voting = 5; //increase 5 times
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const erc20Multi = new Multicaller(network, provider, bep20Abi, {
+    blockTag
+  });
+
+  // returns user's aqua balance ofr their address
+  let score: any = erc20BalanceOfStrategy(
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
+  );
+
+  // returns user's aqua balance in aqua only vault
+  let usersAquaVaultBalances: any = multicall(
+    network,
+    provider,
+    planetFinanceFarmAbi,
+    addresses.map((address: any) => [
+      planetFinanceFarmContractAddress,
+      'stakedWantTokens',
+      ['1', address]
+    ]),
+    { blockTag }
+  );
+
+  //returns user's shares  in aqua auto comp vault
+  let usergAquaBalInAquaInfinityVault: any = multicall(
+    network,
+    provider,
+    aquaInfinityAbi,
+    addresses.map((address: any) => [
+      aquaInfinityAddress,
+      'getUserGtokenBal',
+      [address]
+    ]),
+    { blockTag }
+  );
+
+  // returns user's aqua balance in aqua-bnb vault
+  let usersAquaGammaVaultBalances: any = multicall(
+    network,
+    provider,
+    planetFinanceFarmAbi,
+    addresses.map((address: any) => [
+      gammaFarmAddress,
+      'stakedWantTokens',
+      ['0', address]
+    ]),
+    { blockTag }
+  );
+
+  // returns user's aqua balance in aqua-bnb vault
+  let usersNewAquaBnbVaultBalances: any = multicall(
+    network,
+    provider,
+    planetFinanceFarmAbi,
+    addresses.map((address: any) => [
+      gammaFarmAddress,
+      'stakedWantTokens',
+      ['1', address]
+    ]),
+    { blockTag }
+  );
+
+  //AQUA LENDING
+  let usersAquaInLending: any = multicall(
+    network,
+    provider,
+    aquaLendingAbi,
+    addresses.map((address: any) => [
+      aquaLendingAddress,
+      'getAccountSnapshot',
+      [address]
+    ]),
+    { blockTag }
+  );
+
+  const result = await Promise.all([
+    score,
+    usersAquaVaultBalances,
+    usergAquaBalInAquaInfinityVault,
+    usersAquaGammaVaultBalances,
+    usersNewAquaBnbVaultBalances,
+    usersAquaInLending
```

Comment:
Hi @defininja There is limit of number of number of requests/multicall made from a strategy,


Is there some way to reduce number of calls being sent? and for erc20BalanceOfStrategy you can use `erc20-balance-of` strategy in your space, resulting voting power will be sum of these strategies

Reply:
@ChaituVR - what is the limit on the number of requests and multicalls? Can you please elaborate?

Reply:
Hi @ChaituVR,

I have updated code to match your requirements. Please approve my pull request.
Thanks.

https://github.com/defininja/snapshot-strategies/tree/b380c66d8a87e4879e7b74e108e18bf5e5404fd9

Reply:
hi @ChaituVR - can you please look into it? Thanks.